src/ASM/LR/ASMu3Dmx.C

```diff
@@ -552,18 +558,24 @@ bool ASMu3Dmx::integrate (Integrand& integrand,
 
             // Evaluate the integrand and accumulate element contributions
             fe.detJxW *= 0.125*vol*wg[i]*wg[j]*wg[k];
-            if (!integrand.evalIntMx(*A,fe,time,X))
+            if (!integrand.evalIntMx(*A,fe,time,X)) {
               ok = false;
+              continue;
```

Comment:
Here as well, continue at at the end of the loop block has no effect.

Reply:
removed.